src/SFA.DAS.AssessorService.Application/Handlers/Certificates/UpdateCertificatesPrintStatusHandler.cs

```diff
@@ -70,6 +87,7 @@ private async Task<List<CertificatePrintStatus>> Validate(List<CertificatePrintS
                     .Select(certificatePrintStatus => certificatePrintStatus.Key)
                     .ToList());
 
+            //Scenario 4:  Invalid status in delivery notification file
```

Comment:
This looks like you be able to pass 'Printed' in a delivery notification file - what is stopping that - or will you be checking that in the function app as well?

Reply:
I am checking this in the function app.